spec/data/gobstones/procedure/times_move_fail.yml

```diff
@@ -0,0 +1,27 @@
+:check_head_position: true
+:source: |
+  procedure TimesMove(n, dir) {}
+:subject: TimesMove
+:examples:
+ - :initial_board: |
+     GBB/1.0
+     size 10 10
+     head 0 9
+   :arguments:
+    - 3
+    - south
```

Comment:
This should be `Sur`, because you want to test assertion error and not compilation error here.